features/url-schemes.html

```diff
@@ -16,6 +16,7 @@ <h1>URL Schemes</h1>
  <li><a href="custom://scheme/with.path">custom://scheme/with.path</a> - a custom URL scheme that probably isn't supported</li>
  <li><a href="intent://duckduckgo.com/?q=test#Intent;scheme=https;package=com.duckduckgo.mobile.android;S.browser_fallback_url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.duckduckgo.mobile.android;end">intent://duckduck.com</a> - launches DuckDuckGo Android app using a query parameter or opens the PlayStore</li>
  <li><a href="intent://duckduckgo.com#Intent;scheme=https;package=com.duckduckgo.mobile.android;S.browser_fallback_url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.duckduckgo.mobile.android;end">intent://duckduck.com</a> - launches DuckDuckGo Android app or opens the PlayStore</li>
+ <li><a href="shortcuts://run-shortcut?name="Test%20shortcut">shortcuts://run-shortcut?name="Test%20shortcut</a> - on iOS opens Shortcuts app and triggers execution of "Test shortcut" shortcut</li>
```

Comment:
```suggestion
 <li><a href="shortcuts://run-shortcut?name="Test%20shortcut">shortcuts://run-shortcut?name="Test%20shortcut"</a> - on iOS opens Shortcuts app and triggers execution of "Test shortcut" shortcut</li>
```

Reply:
Thanks for looking at it. Actually the quotes around the name value are not required and there was a stray quote inside href. I will fix it in the following commit.